fitnessllm_dataplatform/task_handler.py

```
@@ -0,0 +1,62 @@
+    def ingest(self, data_source: str) -> None:
+        """Entry point for downloading JSONs from API.
+
+        Args:
+            data_source: Data source to download from.
+
+        Raises:
+            KeyError: If required options or environment variables are missing.
+            ValueError: If data source is not supported.
+        """
+        if data_source not in [member.value for member in FitnessLLMDataSource]:
+            raise ValueError(f"Unsupported data source: {data_source}")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)